.github/workflows/draft-release.yml

```diff
@@ -43,29 +43,38 @@ jobs:
       - name: Checkout syslog-ng source
         uses: actions/checkout@v2
 
-      - name: "DBLD: release"
+      - name: Setup environment
         run: |
-          ./dbld/rules release VERSION=`cat VERSION`
-
-      - name: Store release tarball as artifact
-        uses: actions/upload-artifact@v2
-        with:
-          name: tarball
-          path: dbld/release/${VERSION}/${RELEASE_NAME}.tar.gz
+          . .github/workflows/gh-tools.sh
 
-      - name: Create draft release
-        run: |
           VERSION=`cat VERSION`
           RELEASE_TAG=syslog-ng-$VERSION
           RELEASE_NAME=${RELEASE_TAG}
+          TARBALL_PATH=dbld/release/${VERSION}/${RELEASE_NAME}.tar.gz
+
+          gh_export VERSION RELEASE_TAG RELEASE_NAME TARBALL_PATH
+
+      - name: "DBLD: release"
+        run: |
+          ./dbld/rules release VERSION=${VERSION}
+
+      - name: Create draft release
+        run: |
           sed "1 i${RELEASE_NAME}\n" NEWS.md > /tmp/message
 
           hub release create \
             --draft \
             --file /tmp/message \
-            --attach dbld/release/${VERSION}/${RELEASE_NAME}.tar.gz \
+            --attach ${TARBALL_PATH} \
             ${RELEASE_TAG}
 
+      - name: Store release tarball as artifact
```

Comment:
Can we pleas keep the order of try to upload artifact and if successful create the draft release ?

Reply:
Good idea, sure.

Reply:
Fixed.